### PR TITLE
API Provider Editor UI

### PR DESCRIPTION
**Pull request by kahboom**
Changes:
- `Select Method` UI component, including i18n
- `Review Actions` UI component
- `Set Info` UI component
- `Review Operations` UI component
- Stories for all except `Set Info` (not necessary)

Routes are as follows:

| Path | Description |
| --- | --- |
| `/api-provider/select-method` | Select method: d&d upload, manual, or URL |
| `/api-provider/review-actions` | Review upload details |
| `/api-provider/edit-specification` | Apicurio, edit API specification |
| `/api-provider/set-info` | Set name and description of integration |
| `/api-provider/review-operations` | Review imported and possibly edited operations |

Outstanding:
- Add some navigation
- Add a toolbar in the meantime
- Prepare shared UI components for API Client Connector wizard
- Polish UI components, add better handling for use in pages
- Improve aesthetics

### CODE REVIEW

**Comment by kahboom** [comment]
Story for Select Method, need functions for onclick to automatically select the method:

Screenshot 2019-05-23 16 06 55

Screenshot 2019-05-23 16 07 04

Screenshot 2019-05-23 16 07 20

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1781/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

File: app/ui-react/packages/ui/src/Integration/Editor/apiProvider/ApiProviderSelectMethod.tsx [hunk 1]
```diff
@@ -16,77 +28,40 @@ export class ApiProviderSelectMethod extends React.Component<
   public render() {
     return (
       <>
-        <Text>{this.props.i18nTitle}</Text>
         <Text>{this.props.i18nDescription}</Text>
         <ListView
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Integration/Editor/apiProvider/ApiProviderSelectMethod.tsx]: It just struck me, I don't think using a `ListView` and `ListItems` is appropriate for this here.  It's a pretty standard form so it should really be using `FormGroup` etc.

Comment (kahboom) [reply app/ui-react/packages/ui/src/Integration/Editor/apiProvider/ApiProviderSelectMethod.tsx]: I can do that, let me make some changes

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1818/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1826/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1829/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1849/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1851/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1855/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1857/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1884/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1888/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1890/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Pull request merged**